website/docs/feature-flag-tutorials/use-cases/trunk-based-development.md

```diff
@@ -0,0 +1,222 @@
+-   The release timeline or planned retirement date
+-   Associated Jira tickets or GitHub issues
+-   Relevant documentation or context
+-   [Tagging](/reference/tags) and metadata make it easier to search, filter, and manage your feature flags, especially as the number of flags grows.
+
+Click **Create new tag** in the Unleash Admin UI to add these details to your feature flag.
+
+![Create a tag with relevant data to better organize your feature flags. A modal will pop up to name and save as many tags as you need.](/img/use-case-tbd-tagging.png)
+
+#### Automate Flag Lifecycle Management
+
+As your codebase and development processes mature, consider automating key aspects of feature flag management. This can include:
+
+-   Automated flag archiving and removal: Set rules to automatically archive or remove flags that have been inactive for a certain period, reducing technical debt.
+-   Integration with your CI/CD pipelines: Automatically create, update, or toggle flags as part of your deployment workflows, ensuring flags stay in sync with the codebase.
+-   Reporting and analytics: Generate regular reports on flag usage, performance, and health to proactively identify opportunities for optimization.
+
+Automation helps maintain a clean, well-organized feature flag landscape, even as your organization scales its trunk-based development efforts.
+
+## Control Feature Rollouts
+
+Once a feature is complete, feature flags enable controlled rollouts. You can gradually expose the feature to users, starting with specific groups or environments, without needing to redeploy the code.
+
+For trunk-based development, you can create a gradual rollout strategy to:
+
+-   Determine the percentage of users exposed to the new feature
+-   Determine the percentage of users that get exposed to each version of the feature
+-   Per-customer release
+-   General release
+
+To target users accordingly, let's create an [activation strategy](/reference/activation-strategies). This Unleash concept defines who will be exposed to a particular flag. Unleash comes pre-configured with multiple activation strategies that let you enable a feature only for a specified audience, depending on the parameters under which you would like to release a feature.
+
+![Add a strategy to configure a release process for your flag.](/img/use-case-tbd-add-strategy.png)
+
+The gradual rollout strategy form has multiple fields that control the rollout of your feature. You can name the strategy something relevant to the feature you’re building, but this is an optional field.
+
+Next, configure the rollout percentage so only a certain portion of your users are targeted. For example, you can adjust the dial so that 35% of all users are targeted. The remaining percentage of users will not experience any variation of the new feature. Adjust the rollout dial to set the percentage of users the feature targets, or keep it at 100% to target all users.
```

Comment:
```suggestion
Next, let's create a new activation strategy and configure the rollout percentage so only a certain portion of your users are targeted. For example, you can adjust the dial so that 35% of all users are targeted. The remaining percentage of users will not experience any variation of the new feature. Adjust the rollout dial to set the percentage of users the feature targets, or keep it at 100% to target all users.
```